### PR TITLE
update issue templates

### PR DESCRIPTION
**Pull request by sebastianscatularo**
update issue templates

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+name: 🐞 Feature Request
```
Comment (M-Picco) [inline .github/ISSUE_TEMPLATE/feature.yml:1]: Should this bug emoji be here?

File: .github/ISSUE_TEMPLATE/support.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
-name: 🐞 Development Support Request
-description: File a development support request
-title: "[BUG][DSR] <title>"
-labels: [bug, needs triage, medium]
+name: 🐞 Support Request
```
Comment (M-Picco) [inline .github/ISSUE_TEMPLATE/support.yml:1]: Same here

**Review by M-Picco — APPROVED** [review]
LGTM, save for a possibly misplaced emoji

File: .github/ISSUE_TEMPLATE/support.yml (hunk 2 above)
Comment (sebastianscatularo) [reply .github/ISSUE_TEMPLATE/support.yml]: thank, fixed

File: .github/ISSUE_TEMPLATE/feature.yml (hunk 1 above)
Comment (sebastianscatularo) [reply .github/ISSUE_TEMPLATE/feature.yml]: thanks, fixed

**Pull request merged**